ouroboros-consensus/test-consensus/Test/Consensus/BlockchainTime/Simple.hs

```diff
@@ -13,6 +13,7 @@
 module Test.Consensus.BlockchainTime.Simple (tests) where
 
 import           Control.Exception (SomeException, fromException)
+import           Control.Monad.Class.MonadTimer (MonadDelay (..))
```

Comment:
Remove this import

Reply:
I think this is needed for the instance declaration.

Reply:
What error do you get when you remove it?

Reply:
```
test-consensus/Test/Consensus/BlockchainTime/Simple.hs:324:3: error:
    ‘threadDelay’ is not a (visible) method of class ‘MonadDelay’
    |
324 |   threadDelay d = OverrideDelay $ ReaderT $ \schedule -> do
    |   ^^^^^^^^^^^

test-consensus/Test/Consensus/BlockchainTime/Simple.hs:346:3: error:
    ‘threadDelay’ is not a (visible) method of class ‘MonadDelay’
    |
346 |   threadDelay d = OverrideDelay $ ReaderT $ \_schedule -> threadDelay d
    |   ^^^^^^^^^^^

```

If I import it qualified I get scope to the method, even though I never actually use the module qualified (qualified names are now allowed in bindings positions).